x-pack/legacy/plugins/apm/typings/common.d.ts

```diff
@@ -11,6 +11,8 @@ import {} from '../../reporting/public/components/report_listing';
 // .svg
 import '../../canvas/types/webpack';
 
+type ProcessorEvent = 'transaction' | 'error' | 'metric';
```

Comment:
if we don't export this, does this make it globally available for all of x-pack?

Reply:
Sorry, you mentioned that earlier in this review but forgot about it. You mentioned having it as a constant - where would that be?